spec/fixtures/custom_nginx.template

```diff
@@ -311,18 +238,20 @@ http {
     }
 
     server {
-        listen 55555;
         server_name mock_upstream;
 
+        listen 55555;
+        listen 55556 ssl;
+
+        ssl_certificate ${{SSL_CERT}};
+        ssl_certificate_key ${{SSL_CERT_KEY}};
+        ssl_protocols TLSv1.1 TLSv1.2;
+
         location = / {
-            access_by_lua_block {
-                local mu = require "spec.fixtures.mock_upstream"
-                return mu.filter_access_by_method("GET")
-            }
             content_by_lua_block {
                 local mu = require "spec.fixtures.mock_upstream"
-                return mu.send_json_response({
-                    valid_routes                    = {
+                return mu.send_default_json_response({
```

Comment:
Yes, at the end all endpoints return "the default response". Simpler to implement than adding them case-by-case (this is more obvious on  #2827, where I have refactored this)